config/clients/python/template/client/configuration.mustache

```diff
@@ -14,13 +14,14 @@ class ClientConfiguration(Configuration):
             self,
             api_scheme="https",
             api_host=None,
+            api_url=None,
```

Comment:
Shouldn't `api_url` be at the end for it to not be a breaking change?

Reply:
(caveat here I'm not experience with python)

I thought this, but after looking through the python docs I thought it was safe to include here as we're using keyword arguments not positional arguments. Could totally be wrong though so I'll try and check to determine

Reply:
Unfortunately you can do this in Python:

```py
configuration = ClientConfiguration('http', 'localhost:8080', 'some-store-id', credentials)
```

<img width="973" alt="Screenshot 2024-02-05 at 8 40 53 AM" src="https://github.com/openfga/sdk-generator/assets/536147/a12b8164-45d3-4bbe-a2c5-854b4c025047">

What I would suggest is adding it at the end and add a TODO: so we can move it to first when we drop the deprecated fields in the next few major changes

Reply:
Ah shoot I had overlooked that this was allowed, I'll change the PR to no longer break this.

Reply:
Fixed in 87c5fa862915ed2851cd877658fc280ab1e370a4